specification/translation/Azure.AI.TextTranslation/models-dictionary.tsp

```diff
@@ -1,14 +1,17 @@
 import "@typespec/rest";
 import "@azure-tools/typespec-azure-core";
 import "./models-shared.tsp";
+import "@azure-tools/typespec-client-generator-core";
```

Comment:
We try to keep any customizations that need `"@azure-tools/typespec-client-generator-core"` in a separate `client.tsp` file to avoid coupling the Service definition with the Client representation. For example in the new `client.tsp` you could have

```ts
import "./models-dictionary.tsp"
import "@azure-tools/typespec-client-generator-core";

@@clientName(TextTranslation.BackTranslation.numExamples, "ExamplesCount");

@@clientName(TextTranslation.DictionaryLookupParameters.from, "SourceLanguage");

@@clientName(TextTranslation.DictionaryLookupParameters.to, "TargetLanguage");

@@clientName(TextTranslation.DictionaryExamplesParameters.from,
  "SourceLanguage"
);

@@clientName(TextTranslation.DictionaryExamplesParameters.to, "TargetLanguage");

```

Reply:
Done